### PR TITLE
D8CORE-3498: Added additional contact information field

### PR DESCRIPTION
**Pull request by imonroe**
# READY FOR REVIEW

# Summary
- We add an extra field to events to contain any additional contact information that might be required, and display it on the node page (not in cards or lists)

# Review By (Date)
- 10/17

# Criticality
- normal

# Urgency
- normal

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Rebuild Cache and import config `drush cr ; drush ci`
3. Go to create an event node.
4. Verify there is an "Additional Contact Information" text field.
5. Save an event node with the additional contact information field filled out.
6. verify the information shows up on the node page.

### Site Configuration Sync

- Is there a config:export in this PR that changes the config sync directory? YES

## Front End Validation
- [ ] Design is approved by @ user?
- [ ] HTML validation: Is the markup using the appropriate semantic tags and [passes validation](https://validator.w3.org/nu/)? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Cross-browser testing: Has been performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Automated accessibility: Scans performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Manual accessibility: Manually tested? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket(s)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: config/sync/core.entity_view_display.node.stanford_event.search_indexing.yml [hunk 1]
```diff
@@ -190,6 +195,7 @@ hidden:
   links: true
   search_api_excerpt: true
   stanford_intranet__access: true
+  su_event_contact_info: true
```
Comment (pookmish) [inline config/sync/core.entity_view_display.node.stanford_event.search_indexing.yml:198]: We probably want it to be indexed. This is the display mode that does that for us. Remember to hide the label though.

**Review by pookmish — CHANGES_REQUESTED** [review]
Just a small change and then this is g2g

File: config/sync/core.entity_view_display.node.stanford_event.search_indexing.yml (hunk 1 above)
Comment (imonroe) [reply config/sync/core.entity_view_display.node.stanford_event.search_indexing.yml]: Fixed.

**Review by pookmish — APPROVED** [review]
Thanks.

**Comment by pookmish** [comment]
looks like your tests need a tweak

**Pull request merged**